### PR TITLE
Added tidying branch with new indentation

### PR DESCRIPTION
**Pull request by jessicaantiope**

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -21,46 +21,46 @@
...
+                <h3>The Road to Enlightenment</h3>
+                <p>Littering a dark and dreary road lay the past relics of browser-specific tags, incompatible <abbr title="Document Object Model">DOM</abbr>s, broken <abbr title="Cascading Style Sheets">CSS</abbr> support, and abandoned browsers.</p>
+                <p>We must clear the mind of the past. Web enlightenment has been achieved thanks to the tireless efforts of folk like the <abbr title="World Wide Web Consortium">W3C</abbr>, <abbr title="Web Standards Project">WaSP</abbr>, and the major browser creators.</p>
+                <p>The CSS Zen Garden invites you to relax and meditate on the important lessons of the masters. Begin to see with clarity. Learn to use the time-honored techniques in new and invigorating fashion. Become one with the web.</p>
+            </div>
 
 
         </section>
         <div class="main supporting" id="zen-supporting" role="main">
-        <div class="explanation" id="zen-explanation" role="article">
```
Comment (annika-philipp) [inline pristine-zen-garden.html]: Why did you delete this line?

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -21,46 +21,46 @@
...
-            <p>We ask that you submit original artwork. Please respect copyright laws. Please keep objectionable material to a minimum, and try to incorporate unique and interesting visual themes to your work. We&#8217;re well past the point of needing another garden-related design.</p>
-            <p>This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.</p>
-            <p role="contentinfo">By <a href="http://www.mezzoblue.com/">Dave Shea</a>. Bandwidth graciously donated by <a href="http://www.mediatemple.net/">mediatemple</a>. Now available: <a href="http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">Zen Garden, the book</a>.</p>
+            <div class="explanation" id="zen-explanation" role="article">
```
Comment (annika-philipp) [inline pristine-zen-garden.html]: this is what should have been in line 40 I think

File: pristine-zen-garden.html [hunk 3]
```diff
@@ -21,46 +21,46 @@
...
+                <p>We ask that you submit original artwork. Please respect copyright laws. Please keep objectionable material to a minimum, and try to incorporate unique and interesting visual themes to your work. We&#8217;re well past the point of needing another garden-related design.</p>
+                <p>This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.</p>
+                <p role="contentinfo">By <a href="http://www.mezzoblue.com/">Dave Shea</a>. Bandwidth graciously donated by <a href="http://www.mediatemple.net/">mediatemple</a>. Now available: <a href="http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">Zen Garden, the book</a>.</p>
         </div>
```
Comment (annika-philipp) [inline pristine-zen-garden.html]: you'd want to indent this to be the same level as line 57

File: pristine-zen-garden.html [hunk 4]
```diff
@@ -21,46 +21,46 @@
...
 
         <footer>
```
Comment (annika-philipp) [inline pristine-zen-garden.html]: 66 - 72 same level as line 57

Comment (annika-philipp) [inline pristine-zen-garden.html]: Can't comment in the lines below, but you want to align line 76 with 74 and then indent from there

File: pristine-zen-garden.html (hunk 1 above)
Comment (jessicaantiope) [reply pristine-zen-garden.html]: Not sure actually what happened here.  I think when i modified it i deleted a few things.  I will try and sort them

**Review by annika-philipp — APPROVED** [review]
